packages/inquirer/lib-cjs/inquirer-default.js

```diff
@@ -0,0 +1,3 @@
+const inquirer = require("./inquirer.js").default;
```

Comment:
I'd recommend adding this file to the eslint ignore. It errors on CI now since it's using common.js syntax - and wouldn't compile as-is.